runtime/codert_vm/znathelp.m4

```diff
@@ -296,6 +296,8 @@ define({CINTERP},{
 
 dnl Runtime helpers
 
+DUAL_MODE_HELPER(jitNewValue,1)
+DUAL_MODE_HELPER(jitNewValueNoZeroInit,1)
```

Comment:
Do we need these for AArch64 and ARM as well?

Reply:
Both ARM variants are already in this PR.

Reply:
(now)

Reply:
yes, I updated the arm64nathelp.m4 and armnathelp.m4 helpers